helm/ds/templates/configmap.yaml

```diff
@@ -8,7 +8,7 @@ data:
   DJ_INSTANCE: {{ .Values.instance }}
```

Comment:
should this be quoted also?

Reply:
probably. It theory it is never left blank - which is what caused the issue with the slack url